activestorage/lib/active_storage/engine.rb

```diff
@@ -167,8 +168,10 @@ class Engine < Rails::Engine # :nodoc:
     end
 
     initializer "active_storage.asset" do
-      if Rails.application.config.respond_to?(:assets)
-        Rails.application.config.assets.precompile += %w( activestorage activestorage.esm )
+      config.after_initialize do |app|
+        if Rails.application.config.respond_to?(:assets) && app.config.active_storage.precompile_assets
+          Rails.application.config.assets.precompile += %w( activestorage activestorage.esm )
```

Comment:
Can't we just refer to `app.config` here, rather than the two `Rails.application.config`?

Reply:
✔️ right, that's done.

I wrote it like this to be consistent with what you did in 3fd4185d971b12086dbf45c3bc8f67e21f3ddc94 but it's better like that. For consistency again, I changed the code in Action Cable too.